crates/interop/src/errors.rs

```diff
@@ -31,6 +31,11 @@ pub enum MessageGraphError<E> {
     /// Invalid messages were found
     #[error("Invalid messages found on chains: {0:?}")]
     InvalidMessages(Vec<u64>),
+    /// Missing a [RollupConfig] for a chain ID
+    ///
+    /// [RollupConfig]: maili_registry::RollupConfig
```

Comment:
this should be from `maili_genesis`, I think the type is re-exported from `maili_registry` just fyi

```suggestion
    /// [RollupConfig]: maili_genesis::RollupConfig
```

Reply:
Ah nice - updated